collect_app/src/test/java/org/odk/collect/android/support/MockFormEntryPromptBuilder.java

```diff
@@ -91,4 +91,11 @@ public MockFormEntryPromptBuilder withAnswerDisplayText(String text) {
 
         return this;
     }
+
+    public MockFormEntryPromptBuilder withAnswer(IAnswerData answer) {
+        when(prompt.getAnswerValue()).thenReturn(answer);
+        when(prompt.getAnswerText()).thenCallRealMethod();
```

Comment:
Huh, I didn't know this was possible. I guess this makes the resulting form entry prompt more of a spy.

Reply:
Yeah I'm not a fan of this. Ideally this would just setup a real `FormEntryPrompt` but it's really hard to build one. The `getAnswerText` method definitely feels like a "decorator" that shouldn't really be on the `FormEntryPrompt` as well.